Разметка всех страниц

1ibrarian opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2218579/tasks/5)

1ibrarian committed:
Создал шапку сайта

в шапку сайта добавил логотип и навигационные ссылки

1ibrarian committed:
Добавляет в шапку навигации поиск

1ibrarian committed:
Добавляет в шапку навигации счетчик лайков

1ibrarian committed:
Добавляет в шапку навигации хочу сюда

1ibrarian committed:
оборачивает хочу сюда тегом спан

1ibrarian committed:
добавляет в основной контент картинку с надписями

1ibrarian committed:
добавляет картинку в главный контент

1ibrarian committed:
добавляет в главный контент что такое седона

1ibrarian committed:
исправляет ошибку в использовании тега

1ibrarian committed:
добавляет в главный контент преимущества седоны

1ibrarian committed:
добавляет в главный контент блок заинтересовались

1ibrarian committed:
добавляет ссылку кнопку в блок заинтересовались

1ibrarian committed:
добавляет блок подписки

добавил заголовок подписки его текст поле с вводом e-mail и кнопку подписаться

1ibrarian committed:
добавляет блок футер

в блоке добавил ссылку кнопку вконтакте телеграм и ютуб

1ibrarian committed:
добавляет в блок футер номер телефона и логотип

1ibrarian committed:
добавляет в шапке главной страницы

в теге навигации в слове класс добавил букву а  в значение класса добавил main и в путь картинке добавил #

1ibrarian committed:
добавляет в шапке главной страницы

добавил путь в ссылке главная

1ibrarian committed:
добавляет в шапку главной страницы ссылку якорь

1ibrarian committed:
добавляет в шапку главной страницы ссылку на каталог

1ibrarian committed:
добавляет пропуск между блоками в шапке главной страницы

1ibrarian committed:
добавляет каталог

в шапке каталога изменил тайтл на каталог в логотипе ссылка выходит на главную страницу о седоне ссылка выходит на главную страницу гостиница добавил класс navigation-list-current в navigation-user убрал надпись счетчик лайков на избранное

1ibrarian committed:
Изменяет слово счетчик лайков на избранное в главной страницы навигации

1ibrarian committed:
Изменяет в каталоге шапку навигации

скопировал всю шапку навигации и вложил в тег боди

1ibrarian committed:
Добавляет в каталог главного контента

добавил название и хлебные крошки

1ibrarian committed:
добавляет в каталог фильтры отелей

добавил скрытое название список отлей с фильтрами добавил инфраструктуру со списком где можно выбрать несколько вариантов и тип жилья со списком где один вариант

1ibrarian committed:
Добавляет в каталог главного контента

добавил в каталог фильтра стоимость и кнопки применить и сбросить

1ibrarian committed:
добавляет в каталог фильтров главного контента класс filter

1ibrarian committed:
Добавляет в каталог главной страницы сортировку

добавил  сначала популярное Сначала дешёвое Сначала дорогое

1ibrarian committed:
Добавляет в каталоге главного контента как показать отели

1ibrarian committed:
добавляет в каталог главного контента карточку отеля

1ibrarian committed:
Добавляет исправление

в каталоге главного контента изменил тег head на header  в теге form не было закрывающего тега и заодно проставил пробелы для отделение тегов в <select> с классом select-control поставил кавычки у aria-labe убрал кавычки

1ibrarian committed:
Вложил список отелей в <section class=catalog-hotols

1ibrarian committed:
Удаляет списки отелей из <section class=catalog-hotols>

1ibrarian committed:
Добавляет в каталог остальные карточки отелей

1ibrarian committed:
Добавляет пагинацию для карточек отелей

1ibrarian committed:
Удаляет текст в главной страницы

Удалил текст в  <section class="picture"> и добавил его в <img src  добавил заголовок  h2

1ibrarian committed:
Добавляет заголовок в главной страницы в <section class=advantages>

1ibrarian committed:
Исправляет главную страницу в главном контенте

Между блоками поставил пропуск изменил h4 на h3 и добавил h2 в <section class=interest> изменил h5 на h2 в<section class=newsletter>

1ibrarian committed:
Добавляет в главной страницы в главного контента путь изображения

1ibrarian committed:
Добавляет в футере главой страницы

Добавил h2 номер телефона сделал ссылкой

1ibrarian committed:
Добавляет в каталоге в навигации

Добавил путь у ссылки

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
```
https://validator.w3.org/nu/#textarea
Надо бы по фиксить ошибки типа `error`

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+          </ul>
+        </header>
+
+        <section class="catalog-hotols-filter">
```
`hotols` => `hotels`

Ошибку исправить везде!

senchkim commented on catalog.html:
```diff
@@ -0,0 +1,229 @@
+
+            <fieldset class="catalog-filter-group">
+              <legend class="catalog-filter-title">Стоимость, ₽:</legend>
+              <input class="input-catalog-filter-number" type="number" min="0" value="0">
```
https://up.htmlacademy.ru/htmlcss/35/criteries#HTML-05
У каждого инпута должен быть свой `label`

senchkim commented on index.html:
```diff
@@ -5,8 +5,158 @@
   <body>
+    <header class="main-header">
+      <nav class="main-navigation">
+        <a class="main-header-logo">
```
Это лишнее и можно убрать!
Нет смысла в ссылке

senchkim commented on index.html:
```diff
@@ -5,8 +5,158 @@
+          <li class="advantages-item">
+            <img src="#" class="picture-town" alt="городок Седона">
+          </li>
+          <li class="dvantages-item">
```
`dvantages` => `advantages`

senchkim commented on index.html:
```diff
@@ -5,8 +5,158 @@
+        <h2 class="newsletter-title">Подпишитесь на рассылку</h2>
+        <p class="newsletter-text">Только полезная информация и никакого спама, честное бойскаутское!</p>
+        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <input class="field" placeholder="Ваш e-mail" type="email" name="newsletter-email">
```
Опять же нужен свой `label`

senchkim commented on index.html:
```diff
@@ -5,8 +5,158 @@
+        <h2 class="visually-hidden">Логотип html academy</h2>
+        <a class="logo-html-academy-link" href="https://htmlacademy.ru/">.html academy</a>
+      </section>
+    </footer>
```
В целом работа, отличная - хвалю 👍

1ibrarian committed:
Исправляет ошибки типа error

В главной страницы главного контента в <li class=advantages-item> к изображениям добавил alt

1ibrarian committed:
Исправляет ошибки типа error в главной страницы

В главной страницы главного контента в <li class=advantages-item> к изображениям добавил alt

1ibrarian committed:
Исправляет ошибки типа error

в каталоге в карточках отелей обернул теги a button p в тег li

1ibrarian committed:
Исправляет в каталоге hotols на hotels

1ibrarian committed:
Исправляет ошибки в каталоге

в каталоге главного контента в <fieldset class="catalog-filter-group">
input обернул в label

1ibrarian committed:
Удаляет в главной страницы <a> в <header>

1ibrarian committed:
В главной страницы в преимуществах добавил букву а

1ibrarian committed:
Оборачивает в главной страницы <input> в <label>

1ibrarian committed:
Исправляет в каталоге

В каталоге исправил пробелы вложенности тегов добавил закрывающий тег html

1ibrarian committed:
Добавляет в каталог блок секшен и футер
